src/config.rs

```diff
@@ -1,10 +1,37 @@
+use anyhow::Result;
+use envy::Error;
 use serde::{Deserialize, Serialize};
 
+pub trait FromStringKeyValue: Sized {
```

Comment:
```suggestion
pub trait FromEnvLikeKeyValuePairs: Sized {
```

Reply:
fixed in [65190d2](https://github.com/GiganticMinecraft/SeichiRankingBFF/pull/34/commits/65190d21e3fe07ecd8505838c49a07493b350143)